http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/LeoPublisher.scala

```diff
@@ -122,8 +122,8 @@ final class LeoPublisher[F[_]](
           appQuery.updateStatus(m.appId, AppStatus.Stopping).transaction
         case m: LeoPubsubMessage.StartAppMessage =>
           appQuery.updateStatus(m.appId, AppStatus.Starting).transaction
-        case m: LeoPubsubMessage.UpdateAppMessage =>
-          appQuery.updateStatus(m.appId, AppStatus.Updating).transaction
+        case _: LeoPubsubMessage.UpdateAppMessage =>
```

Comment:
Nit, but we should probably leave a comment here (maybe linking the Jira ticket) to explain why we differ from the other message types